mmocr/apis/inferencers/mmocr_inferencer.py

```diff
@@ -231,6 +252,14 @@ def __call__(
             inputs (InputsType): Inputs for the inferencer. It can be a path
                 to image / image directory, or an array, or a list of these.
             batch_size (int): Batch size. Defaults to 1.
+            det_batch_size (int): Batch size for text detection model.
+                Defaults to None. Overwrite batch_size if it is not None.
+            rec_batch_size (int): Batch size for text recognition model.
+                Defaults to None. Overwrite batch_size if it is not None.
+            kie_batch_size (int): Batch size for KIE model.
+                Defaults to None. Overwrite batch_size if it is not None.
+            chunk_size (int): For chunking inputs. Defaults to None.
+                If it is None, batch_size will be used as chunk_size.
```

Comment:
It's hard to tell the conceptual difference between `chunk_size` and `batch_size` when they are presented together, I think just keeping `batch_size` only may be a better choice - simplicity always goes first.